File: 백준/Bronze/11720. 숫자의 합/숫자의 합.java
import java.io.*;
import java.util.StringTokenizer;
class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int N = Integer.parseInt(br.readLine());
        String n = br.readLine();
        int sum = 0;
        
        for (int i = 0; i < N; i++) {
            sum += n.charAt(i) - '0';
        }
        System.out.print(sum);
    }
}

File: 백준/Bronze/11720. 숫자의 합/숫자의 합.py
# 숫자의 개수 N개 입력받기
# N개의 숫자 입력받기
# N개의 숫자의 합 출력하기
n = int(input())
s = input()
sum = 0
for i in range(n):
  sum += int(s[i])
print(sum)